frontend/src/component/featureTypes/FeatureTypesList.tsx

```diff
@@ -0,0 +1,190 @@
+import { ActionCell } from 'component/common/Table/cells/ActionCell/ActionCell';
+import PermissionIconButton from 'component/common/PermissionIconButton/PermissionIconButton';
+import { ADMIN } from 'component/providers/AccessProvider/permissions';
+import { Edit } from '@mui/icons-material';
+import { useConditionallyHiddenColumns } from 'hooks/useConditionallyHiddenColumns';
+
+export const FeatureTypesList = () => {
+    const { featureTypes, loading } = useFeatureTypes();
+    const theme = useTheme();
+    const isSmallScreen = useMediaQuery(theme.breakpoints.down('md'));
+
+    const columns = useMemo(
+        () => [
+            {
+                accessor: 'id',
+                Cell: ({ value }: { value: string }) => {
+                    const IconComponent = getFeatureTypeIcons(value);
+                    return (
+                        <IconCell
+                            icon={
+                                <IconComponent
+                                    data-loading="true"
+                                    color="action"
+                                />
+                            }
+                        />
+                    );
+                },
+                width: 50,
+                disableSortBy: true,
+            },
+            {
+                Header: 'Name',
+                accessor: 'name',
+                minWidth: 125,
+                Cell: TextCell,
+            },
+            {
+                Header: 'Description',
+                accessor: 'description',
+                width: '80%',
+                Cell: ({ value }: { value: string }) => (
+                    <Typography
+                        component="div"
+                        variant="body2"
+                        color="text.secondary"
+                        lineHeight={2}
+                    >
+                        <TextCell lineClamp={1}>{value}</TextCell>
+                    </Typography>
+                ),
+                disableSortBy: true,
+            },
+            {
+                Header: 'Lifetime',
+                accessor: 'lifetimeDays',
+                Cell: ({ value }: { value: number }) => {
+                    if (value) {
+                        return (
+                            <TextCell>
+                                {value === 1 ? '1 day' : `${value} days`}
+                            </TextCell>
+                        );
+                    }
+
+                    return <TextCell>doesn't expire</TextCell>;
+                },
+                sortInverted: true,
+                minWidth: 150,
+            },
+            {
+                Header: 'Actions',
+                Cell: ({ row: { original: featureType } }: any) => (
+                    <Box sx={theme => ({ padding: theme.spacing(0.5, 0) })}>
+                        <ActionCell>
+                            <PermissionIconButton
```

Comment:
Does this button have any content? What does a screenreader announce for it?

Reply:
It should, because of the tooltip. It's the same as in other places.